Fails to update specific inputs

Failed run: https://github.com/lovesegfault/nix-config/actions/runs/7168554123/job/19517011022

```
Run cpcloud/flake-update-action@v2.0.0
Run cpcloud/flake-dep-info-action@v2.0.10
Run nix flake update darwin
  nix flake update darwin
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    DETERMINATE_NIX_KVM: 0
error: cannot find flake 'flake:darwin' in the flake registries
Error: Process completed with exit code 1.
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -23,6 +23,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: cachix/install-nix-action@v24
         with:
+          install_url: https://releases.nixos.org/nix/nix-2.18.1/install
           extra_nix_config: |
             access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}
       - name: generate flake matrix
@@ -47,6 +48,7 @@ jobs:
         uses: cachix/install-nix-action@v24
         with:
           nix_path: nixpkgs=channel:nixos-unstable-small
+          install_url: https://releases.nixos.org/nix/nix-2.18.1/install
           extra_nix_config: |
             access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}
 

--- a/action.yml
+++ b/action.yml
@@ -49,7 +49,7 @@ runs:
 
     - name: Update ${{ inputs.dependency }}
       shell: bash
-      run: nix flake update ${{ inputs.dependency }}
+      run: nix flake lock --update-input ${{ inputs.dependency }}
 
     - name: Get new commit
       id: get_new_commit

